tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/compose/HelmBridgeExtensionIT.java

```diff
@@ -47,22 +45,16 @@ void withBridgeConfiguration_messageBridged() throws Exception {
         final var ipAddress = hivemqContainerNetwork.get().getIpAddress();
 
         // setup bridge configuration
-        final var resource = getClass().getResource("/bridge-config.xml");
-        final var configMapData = Files.readString(Path.of(Objects.requireNonNull(resource).toURI()));
-        final var bridgeConfigMap = new ConfigMapBuilder().withNewMetadata()
-                .withName("test-bridge-configuration")
-                .endMetadata()
-                .withData(Map.of("config.xml",
-                        configMapData.replace("<host>remote</host>", "<host>" + ipAddress + "</host>")))
-                .build();
-        client.configMaps().inNamespace(namespace).resource(bridgeConfigMap).create();
+        final var bridgeConfiguration =
+                readResourceFile("bridge-config.xml").replace("<host>remote</host>", "<host>" + ipAddress + "</host>");
```

Comment:
Does this actually work? In the testsuite test the bridge could not be established using that IP address. Switching to `localhost` worked fine (and then no replacement is needed).